.github/workflows/Tests.yml

```
@@ -797,19 +797,21 @@ jobs:
         persist-credentials: false
         ref: ${{ needs.check_mats.outputs.build_sha }}
         sparse-checkout: '.github/actions/checkout-and-rebuild'
-    - name: Checkout repository for MATs with ${{ matrix.python-version }}
+    - name: Checkout repository for Style with ${{ matrix.python-version }}
```

actionlint (1.7.7)

property "python-version" is not defined in object type {}

(expression)